src/components/CheckForm/useCheckType.tsx

```diff
@@ -28,5 +28,10 @@ export function useFormCheckTypeGroup(check?: Check) {
     return CHECK_TYPE_OPTIONS.find(({ value }) => value === checkType)?.group;
   }
 
+  // A default group is required for the form to render correctly, so we default to bbe/api-endpoint if no group is provided
+  if (!checkTypeGroup) {
```

Comment:
In what case it wouldn't be provided?

Reply:
Fallback covers unhappy path.
It's provided by router params, meaning that the hook itself could be used when there is no route param setup.

Note: refactored `useFormCheckType` as well, as changing the search param to an incorrect checkType would make the app explode.